server.js

```diff
@@ -1,17 +1,40 @@
 const express = require("express");
 const useSocket = require("socket.io");
+const cors = require("cors");
+
+const sequelize = require("./database");
+
+const auth = require("./api/routes/auth");
 
 const app = express();
 const server = require("http").Server(app);
 const io = useSocket(server, {
   cors: {
-    origin: "http://localhost:3000",
+    origin: "*",
     methods: ["GET", "POST"],
-    allowedHeaders: ["my-custom-header"],
+    allowedHeaders: [
+      "Origin",
+      "Accept",
+      "X-Requested-With",
+      "Content-Type",
+      "Access-Control-Request-Method",
+      "Access-Control-Request-Headers",
+    ],
     credentials: true,
   },
 });
 
-server.listen(5000, () => {
-    console.log("Server runnig at: http://localhost:5000");
-  });
\ No newline at end of file
+app.use(cors());
+app.use(express.json({ extended: false }));
+
+sequelize.authenticate().then(() => {
+  console.log("DB connected");
+});
```

Comment:
Try to move it to `server.listen` callback function to be sure that you will connect to database after the server creation